README.md

```diff
@@ -6,8 +6,9 @@ This firmware uses [Espressif's IDF](https://github.com/espressif/esp-idf)
 
 1. [Download the ESP32 toolchain](http://esp-idf.readthedocs.io/en/v3.1/get-started/index.html#setup-toolchain)
 1. Extract it and add it to your `PATH`: `export PATH=$PATH:<path/to/toolchain>/bin`
-1. Clone **v3.1.3** of the IDF: `git clone --branch v3.1.3 --recursive https://github.com/espressif/esp-idf.git`
+1. Clone **v3.3** of the IDF: `git clone --branch release/v3.3 --recursive https://github.com/espressif/esp-idf.git`
 1. Set the `IDF_PATH` environment variable: `export IDF_PATH=<path/to/idf>`
+1. Run `make menuconfig` then press save and then exit
```

Comment:
@facchinm should we commit a new SDK config to remove this additional step?

Reply:
Absolutely, I wanted to do it but I got stuck in other stuff :)

Reply:
@agdl could you please add the new config to this pull request and remove this new line from the read me. Thanks.